website/docs/components/accordion/partials/specifications/states.md

```diff
@@ -0,0 +1,15 @@
+## States
+
+![Example of accordion states](/assets/components/accordion/accordion-states.png =800x*)
+
+### Contains interactive
+
+![Example of accordion states](/assets/components/accordion/accordion-states-contains-interactive.png =800x*)
+
+### isOpen
+
+![Example of accordion states when is open](/assets/components/accordion/accordion-states-open.png =800x*)
+
+### Contains interactive
```

Comment:
Should this be "isOpen + Contains interactive" ?

Reply:
@cveigt

Reply:
I re-organized the sub-sections, hoping it's more clear now